### PR TITLE
Add Pathfinders corporation card Odyssey

### PR DESCRIPTION
**Pull request by kberg**
Boy things worked out well making this card simple.

### CODE REVIEW

File: src/cards/pathfinders/Odyssey.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        player.defer(card.play(player), Priority.DEFAULT);
+        this.discard(player, card);
+        return undefined;
+      },
+    );
+  }
+
+  private discard(player: Player, card: IProjectCard) {
+    // This is copied straight out of Project Workshop.
+    const cardIndex = player.playedCards.findIndex((c) => c.name === card.name);
```
Comment (bafolts) [inline src/cards/pathfinders/Odyssey.ts:66]: Wish that this threw an error if `cardIndex` were `-1`. Both here and where it was copied. This feels like it belongs in a `player.discardPlayedCard` function versus being copied.

**Review by bafolts — APPROVED** [review]
Creating a `discardPlayedCard` function on `Player` would avoid duplicating the code from `ProjectWorkshop`. Throwing an error if `cardIndex` were `-1` would also help with failing fast.

File: src/cards/pathfinders/Odyssey.ts (hunk 1 above)
Comment (kberg) [reply src/cards/pathfinders/Odyssey.ts:66]: Sigh, yeah, I'll do that in a follow-up.

**Comment by kberg** [comment]
> Creating a `discardPlayedCard` function on `Player` would avoid duplicating the code from `ProjectWorkshop`. Throwing an error if `cardIndex` were `-1` would also help with failing fast.

I'm centralizing the method, but I'm having second thoughts about throwing  an error. The problem can be that it's game breaking - this situation would render a game unplayable. I wish we had a better infrastructure that allowed errors which didn't kill a game.

In the meantime I'm going to `console.log` it in an upcoming PR.

**Pull request merged**